lib/puppet/parser/functions/docker_run_flags.rb

```diff
@@ -5,17 +5,28 @@
 # docker_run_flags.rb
 #
 module Puppet::Parser::Functions
+  newfunction(:'docker::escape', type: :rvalue) do |args|
+    subject = args[0]
+
+    #scope = closure_scope
```

Comment:
Looks like I forgot to remove this comment :clown_face:.  Don't forget to drop it when this is ready for merging.